internal/cache/lru/lru_test.go

```diff
@@ -169,6 +169,41 @@ func (t *CacheTest) TestEraseWhenKeyPresent() {
 	ExpectEq(nil, t.cache.LookUp("burrito"))
 }
 
+func (t *CacheTest) TestEraseCacheWithGivenPrefix() {
```

Comment:
Add a testcase when there is no entry matching prefix

Reply:
Added